<?xml version="1.0" ?>
<launch>
	<node pkg="ros_deep_vision" type="cnn_state_manager.py" name="cnn_state_manager_r2" output="screen">
		<param name="camera_frame" value="/r2/head/asus_depth_optical_frame"/>
		<remap from="/asus/rgb/image_raw" to="/r2/head/asus/rgb/image_raw"/>
		<remap from="/asus/depth/image_raw" to="/r2/head/asus/depth/image_raw"/>
	</node>	
</launch>
